zulip/integrations/hg/zulip_changegroup.py

```diff
@@ -15,6 +16,18 @@
 VERSION = "0.9"
 
 
+def parse_user(user: str) -> str:
+    """Extract the name from user string, or fall back to email or raw string."""
+    # Match: "Name <email@example.com>"
+    match = re.match(r"^(.*)<([^>]+)>$", user)
```

Comment:
Doesn't Python have an `emailaddress` module for parsing these correctly? There's no reason to write custom regexes to parse standard formats.